packages/squidex/src/squidex.ts

```diff
@@ -94,6 +94,12 @@ export class Squidex<T extends { id: string; data: Record<string, unknown> }> {
         };
       }
 
+      // eslint-disable-next-line no-console
```

Comment:
Isn't the logger on the http error handler not working? 
https://github.com/yldio/asap-hub/blob/master/packages/services-common/src/framework/lambda.ts#L70
Old lambdas, still run through this code. If squidex errors are not showing up even with `DEBUG=http` the issue should get fixed there.